File: content/api-documentation/how-to/portfolio/examples/getPositions.go
package main

import (
	"fmt"
	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
)

func init() {
	alpaca.SetBaseUrl("https://paper-api.alpaca.markets")
}

func main() {
	// Get our position in AAPL.
	aapl_position, err := alpaca.GetPosition("AAPL")
	if err != nil {
		fmt.Println("No AAPL position.")
	} else {
		fmt.Printf("AAPL position: %v shares.\n", aapl_position.Qty)
	}

	// Get a list of all of our positions.
	positions, err := alpaca.ListPositions()
	if err != nil {
		fmt.Println("No positions found.")
	} else {
		// Print the quantity of shares for each position.
		for _, position := range positions {
			fmt.Printf("%v shares in %s", position.Qty, position.Symbol)
		}
	}
}

File: content/api-documentation/how-to/portfolio/examples/getPositions.js
const Alpaca = require('@alpacahq/alpaca-trade-api')
const alpaca = new Alpaca()

// Get our position in AAPL.
aaplPosition = alpaca.getPosition('AAPL')

// Get a list of all of our positions.
alpaca.getPositions()
    .then((portfolio) => {
        // Print the quantity of shares for each position.
        portfolio.forEach(function (position) {
            console.log(`${position.qty} shares of ${position.symbol}`)
        })
    })

